readme.md

```diff
@@ -181,4 +188,3 @@ Extension icon made by [Freepik](https://www.freepik.com) from [www.flaticon.com
 ## License
 
 [![CC0](https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
-
```

Comment:
Is an empty line removed here?

Reply:
Yes, likely because the file ended with `\n\n` instead of simply `\n`